.github/workflows/shuffle-tests.yml

```
@@ -15,6 +20,10 @@ env:
   SENTRY_SHUFFLE_TESTS: true
 
 jobs:
+  per-test-coverage:
+    if: ${{ inputs.per-test-coverage == 'true' || github.event_name == 'schedule' }}
```

Equivalent, but with less-surprising semantics:

```suggestion
    if: ${{ inputs.per-test-coverage == 'true' }}
```

---

They aren't equivalent, though. My intent is that the schedule also trigger the per test coverage. Check this run: https://github.com/giovanni-guidini/sentry/actions/runs/6746991658
The per-test tests were skipped.

From the [docs](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs) there's a "notes" section that hints at the inputs being part of the `event`